pilot/docker/Dockerfile.proxy_init

```diff
@@ -1,8 +1,24 @@
-FROM ubuntu:xenial
+FROM ubuntu:xenial as builder
 RUN apt-get update && apt-get upgrade -y && apt-get install -y \
     iproute2 \
     iptables \
  && rm -rf /var/lib/apt/lists/*
 
+FROM gcr.io/distroless/base:debug as distroless
+
```

Comment:
why two `FROM`?  I understand the builder part, and that seems finee, but not sure on this second and third from.  Could you expand?